data/avro-schema/src/main/resources/avro/net/corda/data/virtualnode/VirtualNodeManagementResponse.avsc

```diff
@@ -0,0 +1,25 @@
+{
+  "type": "record",
+  "name": "VirtualNodeManagementResponse",
+  "doc": "The response to an {@link VirtualNodeRequest}.  On success the {@link VirtualNodeResponse#responseType} is {@link VirtualNodeManagementResponseSuccess}, on failure it is {@link VirtualNodeManagementResponseFailure}",
+  "namespace": "net.corda.data.virtualnode",
+  "fields": [
+    {
+      "name": "timestamp",
+      "type": {
+        "type": "long",
+        "logicalType": "timestamp-millis"
+      },
+      "doc": "Time ({@link java.time.Instant}) in milliseconds when the record was created."
+    },
+    {
+      "name": "responseType",
+      "doc": "Result of request, either a success or failure message",
+      "type": [
+        "net.corda.data.virtualnode.VirtualNodeInfo",
```

Comment:
Do we need to include `VirtualNodeInfo` as a possible option for reply? I would much rather stick with `VirtualNodeCreateResponse` and `VirtualNodeManagementResponseFailure` only for now.